src/components/softwareSkills/SoftwareSkill.js

```diff
@@ -10,7 +10,12 @@ export default function SoftwareSkill() {
           {skillsSection.softwareSkills.map((skills,i) => {
             return (
               <li key={i} className="software-skill-inline" name={skills.skillName}>
-                <i className={skills.fontAwesomeClassname}></i>
+                <span
+                      className="iconify"
+                      data-icon={skills.fontAwesomeClassname}
```

Comment:
the `fontAwesomeClassname` should be updated to use iconifyClassname or something similar

Reply:
@kartikcho 
i Have Changed Hover Effect  | Renamed fontawesomeClassname to iconfyClassname and Reverted Edits in REadme.md